src/test/scala/ai/datahunters/md/filter/NotEmptyExtensionFilterSpec.scala

```diff
@@ -0,0 +1,115 @@
+package ai.datahunters.md.filter
+
+import ai.datahunters.md.schema.{BinaryInputSchemaConfig, EmbeddedMetadataSchemaConfig, MetadataTagsSchemaConfig}
+import ai.datahunters.md.{SparkBaseSpec, UnitSpec}
+import org.apache.spark.sql.Row
+import org.apache.spark.sql.types.{DataTypes, StructField, StructType}
+
+class NotEmptyExtensionFilterSpec extends UnitSpec with SparkBaseSpec {
+  import NotEmptyExtensionFilterSpec._
+
+  "A NotEmptyExtensionFilter" should "not pass any row due to lack of allowed extension" in {
+    val filter = new NotEmptyExtensionFilter(Seq[String]())
+    val rdd = sparkSession.sparkContext.parallelize(Data)
+    val df = sparkSession.createDataFrame(rdd, Schema)
+    val outputDF = filter.execute(df)
+    val collectedOutput = outputDF.collect()
+    val outputFields = outputDF.schema.fields.map(_.name)
+    assert( Array(BinaryInputSchemaConfig.IDCol, BinaryInputSchemaConfig.BasePathCol, EmbeddedMetadataSchemaConfig.FileTypeCol,
```

Comment:
@WojtekMin To make it shorter, you can add `import BinaryInputSchemaConfig._` It wil be more clean here and in all cases below